pkg/config/config.go

```diff
@@ -86,3 +88,12 @@ func CreateClientConfig(kubeconfig string, log logr.Logger) (*rest.Config, error
 	logger.V(4).Info("Using specified kubeconfig", "kubeconfig", kubeconfig)
 	return clientcmd.BuildConfigFromFlags("", kubeconfig)
 }
+
+// getKubePolicyNameSpace - setting default KubePolicyNameSpace
+func getKubePolicyNameSpace() string {
+	KubePolicyNamespace := os.Getenv("KYVERNO_NAMESPACE")
```

Comment:
if this method is returning the value, we should set a local variable here instead of the module var

Reply:
Yes, I will do this change.